extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1305,6 +1308,13 @@ export class AmpStoryPage extends AMP.BaseElement {
       return nextElement.id;
     }
 
+    // Link last page to first page if it's on it's own.
+    if (!this.viewer_.hasCapability('swipe')) {
```

Comment:
```suggestion
    if (!this.viewer_.hasCapability('swipe') && firstPage) { return firstPage}
    
    return null;
```

Reply:
Wait this change doesn't make sense... Did you mean to first get the firstPage and then check if it has the capability to swipe? I think it's better to check for the capability first since we can save up a querySelector if we're on a swipeable viewer.